ios/Classes/DRM/FLTEzdrmAssetsLoaderDelegate.h

```diff
@@ -0,0 +1,17 @@
+//
+//  FLTEzdrmAssetsLoaderDelegate.h
+//  better_player
+//
+//  Created by Koldo on 18/05/2021.
+//
+
+#import <Foundation/Foundation.h>
+#import <AVKit/AVKit.h>
+#import <AVFoundation/AVFoundation.h>
+
+
+@interface FLTEzdrmAssetsLoaderDelegate : NSObject
```

Comment:
I think it's good time to change that prefix "FLT". I think this header should be named BetterPlayerEzDRMAssetsLoaderDelegate or BetterPlayerEzDrmAssetsLoaderDelegate.

Reply:
Fixed is this commit 613901e